Analysis/Tutorials/src/histograms.cxx

```diff
@@ -69,7 +70,9 @@ struct CTask {
   void process(aod::Tracks const& tracks)
   {
     for (auto& track : tracks) {
-      ptH->Fill(abs(1.f / track.signed1Pt()));
+      if (track.pt() < pTCut)
+        continue;
+      ptH->Fill(track.pt());
```

Comment:
@jgrosseo @aalkin the problem is that pt is a dynamic column, correct?

Reply:
Yes, that's why it is not in a filter here.